packages/decorators/src/tests/memoize.test.ts

```diff
@@ -39,35 +39,6 @@ describe('memoize()', () => {
       expect(myClass.addOne(1)).toEqual(2);
       expect(spy).toHaveBeenCalledTimes(1);
     });
-
```

Comment:
Can we 🔥 more of these tests?  e.g., `'recalculates the result when first argument changed'` feels like it's just testing what [`function-enhancers` already covers](https://github.com/Shopify/quilt/blob/master/packages/function-enhancers/src/test/memoize.test.ts#L9-L20).